Add: add a Japanese localization file for version 1.20

sharpedmimishee opened the pull request:
I added a Japanese localization file in this PR.

sharpedmimishee commented:
If you could, please apply the translations to 1.21.1 and 1.18.2.

twister716 commented:
素晴らしい翻訳ですが一つ修正の提案があります
「収容物 DISKドライブ」の部分の空白はApplied Energistics 2の翻訳と合わせてない方がいいのではないでしょうか![image](https://github.com/user-attachments/assets/5995e584-901e-436b-95d0-f3aee370afbf)

sharpedmimishee commented:
> 素晴らしい翻訳ですが一つ修正の提案があります 「収容物 DISKドライブ」の部分の空白はApplied Energistics 2の翻訳と合わせてない方がいいのではないでしょうか![image](https://private-user-images.githubusercontent.com/172095629/400152999-5995e584-901e-436b-95d0-f3aee370afbf.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYwMTMwMDAsIm5iZiI6MTczNjAxMjcwMCwicGF0aCI6Ii8xNzIwOTU2MjkvNDAwMTUyOTk5LTU5OTVlNTg0LTkwMWUtNDM2Yi05NWQwLWYzYWVlMzcwYWZiZi5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMDRUMTc0NTAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTZjM2E3Y2QyOWE0MGU0ODg1MjUxMGU1YThhZjQyMTNmMDRiZmNjNzY3N2ExMmJiOWQ2M2YzYmNmMmI0ZjEzNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.RWfimT1fK8C5R_ItHjM7leoV_hVuhisFqKhj3KzoV7Y)

Thank you for telling me. I really appreciate it.
I fixed space letters in translations. 
~~And, I found that "液体セル筐体" is "流体セル筐体" in AE2 repository. Therefore, I fixed "液体DISK筐体" to "流体DISK筐体".~~

Lapis256 commented on src/main/resources/assets/ae2_mega_things/ja_jp.json:
このファイルは古い上、間違った場所にあるので削除してください

sharpedmimishee replied:
I don't know why the file appeared in such a place, but I deleted it. Sorry for taking time. PRs that are too bad should be rejected.

Lapis256 replied:
大丈夫です
一つにまとめてマージするので問題ありません

Lapis256 commented on src/main/resources/assets/ae2_mega_things/lang/ja_jp.json:
```diff
@@ -0,0 +1,34 @@
+{
+    "chat.ae2_mega_things.OnlyEmptyDISKsCanBeDisassembled": "分解ができるのは空のDISKのみです。",
```
ここも[AE2の翻訳](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/c1c95077e2a763a511a3a6a79d3118d538a0f39d/src/main/resources/assets/ae2/lang/ja_jp.json#L242)に寄せたほうがいいですね。

```suggestion
    "chat.ae2_mega_things.OnlyEmptyDISKsCanBeDisassembled": "空のDISKドライブのみが分解できます。",
```

Lapis256 commented:
ありがとうございました
このプルリクエストの内容は以下のバージョンにてリリースされました
- 1.18.2: v0.8.1
- 1.20.1: v1.0.4
- 1.21.1: v2.0.1
